code/__DEFINES/jobs.dm

```diff
@@ -14,45 +14,46 @@
-#define JOB_DISPLAY_ORDER_CARGO_TECHNICIAN 20
-#define JOB_DISPLAY_ORDER_SHAFT_MINER 21
-#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 22
-#define JOB_DISPLAY_ORDER_MEDICAL_DOCTOR 23
-#define JOB_DISPLAY_ORDER_PARAMEDIC 24
-#define JOB_DISPLAY_ORDER_CHEMIST 25
-#define JOB_DISPLAY_ORDER_VIROLOGIST 26
-#define JOB_DISPLAY_ORDER_RESEARCH_DIRECTOR 27
-#define JOB_DISPLAY_ORDER_SCIENTIST 28
-#define JOB_DISPLAY_ORDER_ROBOTICIST 29
-#define JOB_DISPLAY_ORDER_GENETICIST 30
-#define JOB_DISPLAY_ORDER_EXP_CORPS 31 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_HEAD_OF_SECURITY 32
-#define JOB_DISPLAY_ORDER_WARDEN 33
-#define JOB_DISPLAY_ORDER_SECURITY_SERGEANT 34 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_DETECTIVE 35
-#define JOB_DISPLAY_ORDER_SECURITY_OFFICER 36
-#define JOB_DISPLAY_ORDER_JUNIOR_SECURITY_OFFICER 37
-#define JOB_DISPLAY_ORDER_SECURITY_MEDIC 38 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_PRISONER 39
-#define JOB_DISPLAY_ORDER_BRIGOFF 40 //SKYRAT EDIT ADDITON
-#define JOB_DISPLAY_ORDER_NANOTRASEN_REPRESENTATIVE 41 //SKYRAT EDIT ADDITON - CENTRAL ROLES BELOW HERE!
-#define JOB_DISPLAY_ORDER_BLUESHIELD 42
+#define JOB_DISPLAY_ORDER_COMMAND_SECRETARY 4 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_BARTENDER 5
+#define JOB_DISPLAY_ORDER_BOTANIST 6
+#define JOB_DISPLAY_ORDER_COOK 7
+#define JOB_DISPLAY_ORDER_JANITOR 8
+#define JOB_DISPLAY_ORDER_CLOWN 9
+#define JOB_DISPLAY_ORDER_MIME 10
+#define JOB_DISPLAY_ORDER_CURATOR 11
+#define JOB_DISPLAY_ORDER_LAWYER 12
+#define JOB_DISPLAY_ORDER_CHAPLAIN 13
+#define JOB_DISPLAY_ORDER_PSYCHOLOGIST 14
+#define JOB_DISPLAY_ORDER_AI 15
+#define JOB_DISPLAY_ORDER_CYBORG 16
+#define JOB_DISPLAY_ORDER_CHIEF_ENGINEER 17
+#define JOB_DISPLAY_ORDER_STATION_ENGINEER 18
+#define JOB_DISPLAY_ORDER_ATMOSPHERIC_TECHNICIAN 19
+#define JOB_DISPLAY_ORDER_QUARTERMASTER 20
+#define JOB_DISPLAY_ORDER_CARGO_TECHNICIAN 21
+#define JOB_DISPLAY_ORDER_SHAFT_MINER 22
+#define JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER 23
+#define JOB_DISPLAY_ORDER_MEDICAL_DOCTOR 24
+#define JOB_DISPLAY_ORDER_PARAMEDIC 25
+#define JOB_DISPLAY_ORDER_CHEMIST 26
+#define JOB_DISPLAY_ORDER_VIROLOGIST 27
+#define JOB_DISPLAY_ORDER_RESEARCH_DIRECTOR 28
+#define JOB_DISPLAY_ORDER_SCIENTIST 29
+#define JOB_DISPLAY_ORDER_ROBOTICIST 30
+#define JOB_DISPLAY_ORDER_GENETICIST 31
+#define JOB_DISPLAY_ORDER_EXP_CORPS 32 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_HEAD_OF_SECURITY 33
+#define JOB_DISPLAY_ORDER_WARDEN 34
+#define JOB_DISPLAY_ORDER_SECURITY_SERGEANT 35 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_DETECTIVE 36
+#define JOB_DISPLAY_ORDER_SECURITY_OFFICER 37
+#define JOB_DISPLAY_ORDER_JUNIOR_SECURITY_OFFICER 38
+#define JOB_DISPLAY_ORDER_SECURITY_MEDIC 39 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_PRISONER 40
+#define JOB_DISPLAY_ORDER_BRIGOFF 41 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_NANOTRASEN_REPRESENTATIVE 42 //SKYRAT EDIT ADDITON - CENTRAL ROLES BELOW HERE!
+#define JOB_DISPLAY_ORDER_BLUESHIELD 43
```

Comment:
Instead of massively changing every define, which is incredibly unmodular, tack it onto the end